takeshape/penny
# Use branches in Vercel build

### Test Plan (Steps to test):

1. Make a branch of this branch
2. Locally make a change to the schema and accompanying code. For example you could rename `addMembers` -- then you'd be able to verify if the renamed mutation is used by using the newsletter subscribe form and checking the mutation name in the browser dev tools.
3. Push up your branch
4. Make sure the deploy succeeds at https://vercel.com/takeshape/penny/deployments
5. At the end of the deployment grab the URL from the assigning domains step
6. Make sure your change is reflected in your preview env
7. Make sure your change is not reflected at https://penny-demo.takeshape.io/
8. Review the `schema.json` carefully. I moved over the production schema to the repo and I added a few comments about some details.

### Checklist:
